src/deepsparse/pipeline.py

```diff
@@ -802,3 +805,14 @@ def zero_shot_text_classification_pipeline(*args, **kwargs) -> "Pipeline":
     is returned depends on the value of the passed model_scheme argument.
     """
     return Pipeline.create("zero_shot_text_classification", *args, **kwargs)
+
+
+def _check_model_path_arg(*args, **kwargs):
+    if args:
+        if len(args) > 1 or "model_path" in kwargs or "model" in kwargs:
```

Comment:
why not have the functions accept `model_path` as an arg directly? then python would raise an error automatically if kwargs also have model_path, we'll still have to check for `model` in kwargs though? Thoughts?


for example changing:

```python
def chat_pipeline(*args, **kwargs) -> "Pipeline":
```

To
```python
def chat_pipeline(model_path, **kwargs) -> "Pipeline":
```

Reply:
Can't just have `model_path` as we also supprt `model` so seems cleaner to just accept args and parse?